ddtrace/contrib/gevent/greenlet.py

```diff
@@ -43,11 +43,17 @@ def __init__(self, *args, **kwargs):
 
 
 if issubclass(gpool.IMap, gpool.IMapUnordered):
-    # For gevent >=1.1, IMap derives from IMapUnordered, so we derive
-    # from TracedIMapUnordered and get tracing that way
-    class TracedIMap(gpool.IMap, TracedIMapUnordered):
-        def __init__(self, *args, **kwargs):
-            super(TracedIMap, self).__init__(*args, **kwargs)
+    if hasattr(gevent, '_imap'):
```

Comment:
It might be nicer/more clear if we could do something like:

```python
if gevent.__version__ >= '1.3.0':
    # patch gevent 1.3
else:
    # patch gevent 1.1 & 1.2
```

Reply:
It is not a hard requirement to make this change. Your call on if making a change like this is worth the effort or not.